src/crypto/crypto_util.cc

```diff
@@ -285,6 +243,64 @@ bool CryptoErrorStore::Empty() const {
   return errors_.empty();
 }
 
+v8::MaybeLocal<v8::Value> cryptoErrorListToException(
+    Environment* env, const ncrypto::CryptoErrorList& errors) {
+  // The CryptoErrorList contains a listing of zero or more errors.
+  // If there are no errors, it is likely a bug but we will return
+  // an error anyway.
+  if (errors.empty()) {
+    return Exception::Error(FIXED_ONE_BYTE_STRING(env->isolate(), "Ok"));
+  }
+
+  // The last error in the list is the one that will be used as the
+  // error message. All other errors will be added to the .opensslErrorStack
+  // property. We know there has to be at least one error in the list at
+  // this point.
+  auto& last = errors.peek_back();
+  Local<String> message;
+  if (!String::NewFromUtf8(
+           env->isolate(), last.data(), NewStringType::kNormal, last.size())
+           .ToLocal(&message)) {
+    return MaybeLocal<Value>();
+  }
+
+  Local<Value> exception = Exception::Error(message);
+  CHECK(!exception.IsEmpty());
+
+  if (errors.size() > 1) {
+    CHECK(exception->IsObject());
+    Local<Object> exception_obj = exception.As<Object>();
+    std::vector<Local<Value>> stack(errors.size() - 1);
+
+    // Iterate over all but the last error in the list.
+    auto current = errors.begin();
+    auto last = errors.end();
+    last--;
+    while (current != last) {
+      Local<String> error;
+      if (!String::NewFromUtf8(env->isolate(),
+                               current->data(),
+                               NewStringType::kNormal,
+                               current->size())
+               .ToLocal(&error)) {
+        return MaybeLocal<Value>();
```

Comment:
I feel like you don't have `clang-tidy` enabled on your IDE :-)
```suggestion
        return {};
```